ledger/src/sigverify_shreds.rs

```diff
@@ -850,7 +850,7 @@ mod tests {
         .collect();
         assert_eq!(
             shreds.len(),
-            packets.iter().map(PacketBatch::len).sum::<usize>()
+            packets.iter().map(|batch| batch.len()).sum::<usize>()
```

Comment:
Was this change strictly necessary in order to compile ? I think it is fine to make, more so curious

Reply:
Yes, exactly, that change is necessary in order to compile.

The reason is that the `len()` method doesn't belong to `PacketBatch` anymore, but rather to `Vec<Packet>`, to which `PacketBatch` derefs. I think something like `.map(Vec<Packet>::len)` would work as well, but I think it would be a bit confusing.